birdhouse/docker-compose.yml

```diff
@@ -297,6 +297,7 @@ services:
       MAGPIE_PROVIDERS_CONFIG_PATH: "/opt/local/src/magpie/config/providers"
       MAGPIE_PERMISSIONS_CONFIG_PATH: "/opt/local/src/magpie/config/permissions"
       MAGPIE_POSTGRES_HOST: postgres-magpie
+      MAGPIE_PORT: ${MAGPIE_PORT}
```

Comment:
@dbyrns This was not needed with previous Magpie 3.5.1, so probably not be needed now either.  Anyways, there's nothing in `default.env` and `env.local.example` with this var so what would be the value of this `MAGPIE_PORT`?

Reply:
The default value has been dropped in Magpie, but you're right no need to add a variable for that. I set it directly as for others birds.